x-pack/plugins/apm/public/components/app/correlations/context_popover/top_values.tsx

```diff
@@ -67,103 +231,65 @@ export function TopValues({ topValueStats, onAddFilter, fieldValue }: Props) {
-                              : value.key,
-                          include: true,
-                        });
-                      }}
-                      aria-label={i18n.translate(
-                        'xpack.apm.correlations.fieldContextPopover.addFilterAriaLabel',
-                        {
-                          defaultMessage: 'Filter for {fieldName}: "{value}"',
-                          values: { fieldName, value: value.key },
-                        }
-                      )}
-                      data-test-subj={`apmFieldContextTopValuesAddFilterButton-${value.key}-${value.key}`}
-                      style={{
-                        minHeight: 'auto',
-                        width: theme.eui.euiSizeL,
-                        paddingRight: 2,
-                        paddingLeft: 2,
-                        paddingTop: 0,
-                        paddingBottom: 0,
-                      }}
-                    />
-                    <EuiButtonIcon
-                      iconSize="s"
-                      iconType="minusInCircle"
-                      onClick={() => {
-                        onAddFilter({
-                          fieldName,
-                          fieldValue:
-                            typeof value.key === 'number'
-                              ? value.key.toString()
-                              : value.key,
-                          include: false,
-                        });
-                      }}
-                      aria-label={i18n.translate(
-                        'xpack.apm.correlations.fieldContextPopover.removeFilterAriaLabel',
-                        {
-                          defaultMessage: 'Filter out {fieldName}: "{value}"',
-                          values: { fieldName, value: value.key },
-                        }
-                      )}
-                      data-test-subj={`apmFieldContextTopValuesExcludeFilterButton-${value.key}-${value.key}`}
-                      style={{
-                        minHeight: 'auto',
-                        width: theme.eui.euiSizeL,
-                        paddingTop: 0,
-                        paddingBottom: 0,
-                        paddingRight: 2,
-                        paddingLeft: 2,
-                      }}
-                    />
-                  </>
-                ) : null}
-              </EuiFlexGroup>
+              <TopValue
+                value={value}
+                barColor={barColor}
+                valueText={valueText}
+                onAddFilter={onAddFilter}
+                progressBarMax={progressBarMax}
+                isHighlighted={isHighlighted}
+                fieldName={fieldName}
+              />
             </>
           );
         })}
+
+      {Array.isArray(fieldValueStats?.topValues) && (
+        <>
+          <EuiHorizontalRule margin="s" />
+          <EuiSpacer size="s" />
+          <EuiText size="xs">
+            <FormattedMessage
```

Comment:
Most places in APM we just use `i18n.translate` directly instead of using the `FormattedMessage` component.

Reply:
Updated here https://github.com/elastic/kibana/pull/118069/commits/58d8bcc2f17926342d614fbf4049f2d2cfcc0749